src/pages/Dashboard/scenes/Emails/Emails.tsx

```diff
@@ -64,19 +69,70 @@ const Emails: React.FC = () => {
   };
 
   const handleSelectChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    let selectedMentors: Mentor[] = [];
+    let selectedMentees: Mentee[] = [];
     switch (e.target.value) {
       case 'allMentors':
-        break;
-      case 'allMentees':
+        selectedMentors = mentors ?? [];
         break;
       case 'acceptedMentors':
+        selectedMentors =
+          mentors?.filter(
+            (mentor) => mentor.state === ApplicationStatus.APPROVED
+          ) ?? [];
```

Comment:
```suggestion
          mentors?.filter(
            (mentor) => mentor.state === ApplicationStatus.APPROVED
          );
```

Do we need nullish coelsion?

Reply:
Because sometimes we will have empty array, right? And admin also can enter email addresses manually after fetching relevant email addresses.

Reply:
![image](https://github.com/sef-global/scholarx-frontend/assets/48531182/97ba7e38-d3b3-4c33-ae79-d5c46c5d2144)